### PR TITLE
Feature/family branch

### PR DESCRIPTION
**Pull request by nforman91**
## Description

This PR includes the Current Courses Details component, adds state to the Parent Family Home component, and styling for both the Create New Student and Current Courses Details components. Also, the commented-out code in the Parent Family folder has been removed.

Fixes # (issue)

[Trello card](https://trello.com/c/MzXpqW47)
[Loom video](https://www.loom.com/share/17bcadf008e94b879868103a0adb184d)

Please paste a link here of a quick loom video walking through what you did in your PR .

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/ParentFamily/CreateNewStudent.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
```
Comment (RoseGabriely) [inline src/components/pages/ParentFamily/CreateNewStudent.js:1]: Unused import here you can remove!

File: src/components/pages/ParentFamily/CreateNewStudent.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  const [newStudentInfo, setNewStudentInfo] = useState();
+
+  const handleChange = e => {
+    e.preventDefault();
+    setNewStudentInfo({
+      ...newStudentInfo,
+      [e.target.name]: e.target.value,
+    });
+  };
+  console.log(newStudentInfo);
```
Comment (RoseGabriely) [inline src/components/pages/ParentFamily/CreateNewStudent.js:15]: Make sure to clean up console logs before merge

File: src/components/pages/ParentFamily/ParentFamilyHome.js [hunk 3]
```diff
@@ -1,9 +1,34 @@
...
+    const token = JSON.parse(localStorage.getItem(`okta-token-storage`));
+    const config = {
+      headers: { Authorization: `Bearer ${token.idToken.value}` },
+    };
+    axios
+      .get(`https://coder-heroes-api.herokuapp.com/parent/1/children`, config)
+      .then(res => {
+        const familyData = res.data;
+        setStudentInfo(familyData);
+        console.log(res.data);
```
Comment (RoseGabriely) [inline src/components/pages/ParentFamily/ParentFamilyHome.js:25]: clean up console.logs!

File: src/components/pages/ParentFamily/ParentFamilyHome.js [hunk 4]
```diff
@@ -1,9 +1,34 @@
...
+      })
+      .catch(err => {
+        console.log(`error fetching axios call`);
```
Comment (RoseGabriely) [inline src/components/pages/ParentFamily/ParentFamilyHome.js:28]: would it be better to take advantage of whatever is being returned in the "err" object or do you feel hard-coding the error is better here?

File: src/components/pages/ParentFamily/CurrentCoursesDetails.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Table, Tag, Space } from 'antd';
```
Comment (RoseGabriely) [inline src/components/pages/ParentFamily/CurrentCoursesDetails.js:2]: address linting errors in regards to unused imports

**Review by RoseGabriely — CHANGES_REQUESTED** [review]
Awesome additions here. Make sure to address the linting errors and keep console logs out of merged code. Also would love to see the redux implemented for this in the near future.

**Review by RoseGabriely — APPROVED** [review]

**Review by ElchinNasirov — APPROVED** [review]

File: src/components/pages/ParentFamily/ParentFamilyHome.js (hunk 4 above)
Comment (paulstgermain) [reply src/components/pages/ParentFamily/ParentFamilyHome.js:26]: I actually tend to agree that passing the values contained in that error would be better error handling - let's look out for things like this moving forward!

**Pull request merged**